lib/net/ssh/transport/algorithms.rb

```diff
@@ -173,6 +182,23 @@ def rekey!
           send_kexinit
         end
 
+        def accept_ext_info(packet)
+          info { "got EXT_INFO from server" }
```

Comment:
This info comment might make more sense using the same style as near line 419 in this file? i.e. 'debug do'...

Reply:
I guess there are other instances of info logging in the repo, feel free to ignore this.